AcademyResidentInformationApi.Tests/IntegrationTests.cs

```diff
@@ -20,6 +26,11 @@ public void OneTimeSetUp()
             var npgsqlCommand = _connection.CreateCommand();
             npgsqlCommand.CommandText = "SET deadlock_timeout TO 30";
             npgsqlCommand.ExecuteNonQuery();
+
+            var builder = new DbContextOptionsBuilder();
+            builder.UseNpgsql(_connection);
+            AcademyContext = new AcademyContext(builder.Options);
+            AcademyContext.Database.EnsureCreated();
```

Comment:
I would add these two lines to the `[SetUp]` method and make builder a private field. Because EF will track changes within a context, making a new one before each test will keep the tests independent. (This is what was causing a lot of the failing tests)

Reply:
👍